services/movie.py

```diff
@@ -0,0 +1,42 @@
+from typing import Optional, List
+
+from db.models import Movie, Genre, Actor
+
+
+def get_movies(genres_ids: Optional[List[int]] = None,
+               actors_ids: Optional[List[int]] = None) -> None:
```

Comment:
The return type hint for `get_movies` is incorrect. This function returns a QuerySet of `Movie` objects, not `None`. Consider changing the return type hint to `QuerySet[Movie]`.